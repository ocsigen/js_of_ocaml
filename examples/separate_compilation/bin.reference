hello world!
hello
Lib1_mod2 was linked
Lib1_mod1 was linked
