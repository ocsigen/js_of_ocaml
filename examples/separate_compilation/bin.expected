hello world!
hello
Lib1_mod1 was linked
